Sources/ArcGISToolkit/Components/Compass/Compass.swift

```diff
@@ -41,14 +44,13 @@ public struct Compass: View {
     /// direction toward true East, etc.).
     /// - Parameters:
     ///   - heading: The heading of the compass.
-    ///   - autoHide: A Boolean value that determines whether the compass
-    ///   automatically hides itself when the heading is `0`.
+    ///   - action: An action to perform when the compass is tapped.
```

Comment:
some doc about when you need to specify this action might be helpful to the user.  For example, does the compass set the heading to zero and call the action if I have an action set?  What is the behavior when no action is set?

Reply:
The default is to rotate the map view to zero, but without animation.

We'll update the doc.